content/en/tracing/setup/nginx.md

```diff
@@ -96,6 +99,34 @@ Complete examples:
 
 After completing this configuration, HTTP requests to NGINX will initiate and propagate Datadog traces, and will appear in the APM UI.
 
+## NGINX Ingress Controller
+
+The Kubernetes [ingress-nginx](https://github.com/kubernetes/ingress-nginx) controller versions 0.23.0+ include the NGINX plugin for OpenTracing.
+
+To enable this plugin, create or edit a ConfigMap to set `enable-opentracing: "true"` and the `datadog-collector-host` to which traces should be sent.
+The name of the ConfigMap will be cited explicitly by the nginx-ingress controller container's command line argument, defaulting to `--configmap=$(POD_NAMESPACE)/nginx-configuration`.
+If ingress-nginx was installed via helm chart, this ConfigMap will be named like `Release-Name-nginx-ingress-controller`.
+
+The ingress controller manages both the `nginx.conf` and `/etc/nginx/opentracing.json` files. Tracing is enabled for all Locations.
+
+```yaml
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: nginx-configuration
+  namespace: ingress-nginx
+  labels:
+    app.kubernetes.io/name: ingress-nginx
+    app.kubernetes.io/part-of: ingress-nginx
+data:
+  enable-opentracing: "true"
+  datadog-collector-host: datadogd.monitoring.svc.cluster.local
```

Comment:
This would require a service in namespace `monitoring` named `datadogd`, correct?
I'll have to check if that's appropriate for the existing documented ways of setting up the datadog agent on Kubernetes.

Reply:
It would. I'm open to other example hostnames. I'm not sure whether a standard example exists.

This parameter does support environment variable substitution. If using Datadog as a Daemonset this could be set to the the node's $HOST_IP. This is described in https://kubernetes.github.io/ingress-nginx/user-guide/third-party-addons/opentracing/ but I thought that overly complicated the example.

Reply:
This is great work @alanjcastonguay. Thanks again for your contribution. I'll merge this for now, and plan for adding a working example of the `$HOST_IP`, consistent with the rest of our docs.

Reply:
Sounds good to me.